stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -23,7 +23,7 @@ const SecuritiesTable = () => {
       })
       .catch(() => {
         setIsLoading(false);
-        setErrorMessage(timeoutMessage);
+        setErrorMessage([timeoutMessage]);
```

Comment:
If you're setting an array of messages here then shouldn't it be called `errorMessages`?

Reply:
Good point. Changed now.